response_operations_ui/controllers/case_controller.py

```diff
@@ -9,25 +9,21 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def get_available_case_group_statuses(short_name, period, ru_ref):
-    logger.debug('Retrieving case group status', short_name=short_name, period=period, ru_ref=ru_ref)
-    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/case/status/{short_name}/{period}/{ru_ref}'
-    response = requests.get(url)
-    if response.status_code != 200:
-        raise ApiError(response)
-
-    logger.debug('Successfully retrieved case group status', short_name=short_name, period=period, ru_ref=ru_ref)
-    return response.json()
-
+def update_case_group_status(collection_exercise_id, ru_ref, case_group_event):
+    logger.debug('Updating status', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref,
+                 case_group_event=case_group_event)
+    url = f'{app.config["CASE_URL"]}/casegroups/transitions/{collection_exercise_id}/{ru_ref}'
+    response = requests.put(url, auth=app.config['CASE_AUTH'], json={'event': case_group_event})
 
-def update_case_group_statuses(short_name, period, ru_ref, event):
-    logger.debug('Updating case group status', short_name=short_name, period=period, ru_ref=ru_ref)
-    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/case/status/{short_name}/{period}/{ru_ref}'
-    response = requests.post(url, json={'event': event})
-    if response.status_code != 200:
+    try:
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        logger.exception('Error updating status', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref,
+                         case_group_event=case_group_event)
         raise ApiError(response)
 
-    logger.debug('Successfully updated case group status', short_name=short_name, period=period, ru_ref=ru_ref)
+    logger.debug('Successfully updated status', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref,
```

Comment:
Is this log line talking about a specific case or a case group?

Reply:
It is but we don't have access to a case group id here
A case group is in a one to one relationship with a combination of ru_ref and collection exercise so we would have sight the case group from these details anyway

Reply:
Will change the wording to `Successfully updated case group status` though

Reply:
That's all I wanted to clarify was the logging wording 👍